config/sc-config.yaml

```
@@ -450,6 +450,18 @@ prometheus:
       percent: 80
       count: 1638400
     exclude: []
+    # Custom per-bucket alerts
+    buckets: []
+    #  - name: <cluster>-thanos # Also add name to exclude from regular alerts.
```

With the most recent changes you do not need to add bucket names to the exclude list right?
```suggestion
    #  - name: <cluster>-thanos # This gets excluded from regular object storage alerts
```

---

True I forgot about that :+1: